docs/system/system-metrics.md

```diff
@@ -15,154 +15,689 @@ instruments not explicitly defined in the specification.
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `system.network.connections` | UpDownCounter | `{connection}` |  |
+<!-- endsemconv -->
+
+<!-- semconv metric.system.network.connections(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| [`network.transport`](../general/attributes.md) | string | [OSI Transport Layer](https://osi-model.com/transport-layer/) or [Inter-process Communication method](https://en.wikipedia.org/wiki/Inter-process_communication). The value SHOULD be normalized to lowercase. | `tcp`; `udp` | Recommended |
+| `system.device` | string | The device identifier | `(identifier)` | Recommended |
+| `system.network.state` | string | A stateless protocol MUST NOT set this attribute | `close_wait` | Recommended |
+
+`network.transport` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+
+| Value  | Description |
+|---|---|
+| `tcp` | TCP |
+| `udp` | UDP |
+| `pipe` | Named or anonymous pipe. See note below. |
+| `unix` | Unix domain socket |
+
+`system.network.state` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `close` | close |
+| `close_wait` | close_wait |
+| `closing` | closing |
+| `delete` | delete |
+| `established` | established |
+| `fin_wait_1` | fin_wait_1 |
+| `fin_wait_2` | fin_wait_2 |
+| `last_ack` | last_ack |
+| `listen` | listen |
+| `syn_recv` | syn_recv |
+| `syn_sent` | syn_sent |
+| `time_wait` | time_wait |
+<!-- endsemconv -->
+## Aggregate System Process Metrics
+
+**Description:** System level aggregate process metrics captured under the namespace `system.processes`.
+For metrics at the individual process level, see [process metrics](process-metrics.md).
+
+### Metric: `system.processes.count`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.system.processes.count(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `system.processes.count` | UpDownCounter | `{process}` | Total number of processes in each state |
+<!-- endsemconv -->
+
+<!-- semconv metric.system.processes.count(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `system.processes.status` | string | The process state, e.g., [Linux Process State Codes](https://man7.org/linux/man-pages/man1/ps.1.html#PROCESS_STATE_CODES) | `running` | Recommended |
+
+`system.processes.status` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+
+| Value  | Description |
+|---|---|
+| `running` | running |
+| `sleeping` | sleeping |
+| `stopped` | stopped |
+| `defunct` | defunct |
```

Comment:
@frzifus regarding https://github.com/open-telemetry/semantic-conventions/pull/89#issuecomment-1662262781:

It does not change the attribute _values_ ("running", ...) but the attribute _key_ from `status` to `system.processes.status` as per https://github.com/open-telemetry/semantic-conventions/issues/61.